### ISSUE TITLE
Give pi-314159 write access on oqs-demos

### ISSUE THREAD

**Issue by dstebila** [open-quantum-safe/tsc#65]
@pi-314159 has been maintaining oqs-boringssl and has been working to update one of the demos as in https://github.com/open-quantum-safe/oqs-demos/pull/291.  Can we give them write access on oqs-demos?

@ryjones Given that #62 is changing the config file I wasn't sure where to make an actual PR for this, whether you would want it on the existing config file or after #62 lands.

**Comment by ryjones** [comment]
Please make a new PR

**Comment by dstebila** [comment]
> Please make a new PR

Okay, I made #67
